Libraries/Components/AppleTV/TVViewPropTypes.js

```diff
@@ -39,6 +39,9 @@ var TVViewPropTypes = {
      * shiftDistanceY: Defaults to 2.0.
      * tiltAngle: Defaults to 0.05.
      * magnification: Defaults to 1.0.
+     * pressMagnification: Defaults to 1.0.
```

Comment:
Let's document these in https://github.com/facebook/react-native-website as well, since those are the docs that are displayed on the website itself.

Reply:
Thanks for the information I've created a pull request here https://github.com/facebook/react-native-website/pull/176